### ISSUE TITLE
Cursor vanishes after opening inventory and doing things

### ISSUE THREAD

**Issue by Dragonissa** [Metacinnabar/OptimizedCursor#1]
Nothing useful in the log except the following:

Running a couple of mods but nothing that touches the cursor.

```
[00:22:20.431] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
[00:22:20.446] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
[00:22:20.464] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
[00:22:20.480] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
[00:22:20.505] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
[00:22:20.520] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
[00:22:20.547] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
[00:22:20.563] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
[00:22:20.580] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
[00:22:20.596] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
[00:22:20.613] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
[00:22:20.630] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
[00:22:20.647] [Main Thread/ERROR] [OptimizedCursor]: SDL_SetCursor threw an error: CreateIconIndirect(): The parameter is incorrect.
```

**Comment by Dragonissa** [comment]
Appears to be fixed with the rewritten version. Which isnt out yet but can be tested on the discord.

**Comment by Metacinnabar** [comment]
Should be fixed in commit 87d94ecd5abc551cd64eb9ea48cc02f036243f02.
